### PR TITLE
[ATL-2018] added event, dates, location, and organizers for Atlanta 2018

### PR DESCRIPTION
**Pull request by cacorriere**
inital PR for Atlanta 2018.

### CODE REVIEW

File: content/events/2017-atlanta/program/sponsors.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+speakers = []
+youtube = ""
+vimeo = ""
+speakerdeck = ""
+slideshare = ""
+slides = ""
++++
+
+
+redhat google mailchimp
```
Comment (bridgetkromhout) [inline content/events/2017-atlanta/program/sponsors.md]: This isn't the usual way to list sponsors - normally they go in the yaml file.

Comment (cacorriere) [reply content/events/2017-atlanta/program/sponsors.md]: I just ran the create event script. Should I delete the file?

Comment (cacorriere) [reply content/events/2017-atlanta/program/sponsors.md]: it's gone. I never added it before but didn't delete it off my computer. It finally got caught in a git add .

**Comment by bridgetkromhout** [comment]
Whomever has time to merge this, please also sync up Slack and the email list. Thanks.

**Pull request merged**